infrastructure/frontend/frontend.template.yml

```diff
@@ -457,174 +460,12 @@ Resources:
             Principal:
               AWS: !FindInMap [ ElasticLoadBalancer, !Ref AWS::Region, AccountID ]
 
-  LoadBalancerWebACL:
```

Comment:
Noticed this has WAF configuration in this that is being deleted, is there any reason for this?

Reply:
It has been moved to separate stack and referenced here. Please see change starting from line 463:

```
  LoadBalancerWebACLAssociation:
    Type: AWS::WAFv2::WebACLAssociation
    Properties:
      WebACLArn: !GetAtt LoadBalancerWebACL.Arn.    <removed>
      WebACLArn: !ImportValue product-pages-waf-acl-arn <added>
```